pkg/scripts/os.go

```diff
@@ -313,18 +313,27 @@ sudo systemctl restart kubelet
 source /etc/kubeone/proxy-env
 
 {{ template "detect-host-cpu-architecture" }}
-{{ template "docker-daemon-config" . }}
 {{ template "sysctl-k8s" }}
 {{ template "journald-config" }}
 
-sudo systemctl restart docker
-
 sudo mkdir -p /opt/cni/bin /etc/kubernetes/pki /etc/kubernetes/manifests
 curl -L "https://github.com/containernetworking/plugins/releases/download/v{{ .KUBERNETES_CNI_VERSION }}/cni-plugins-linux-${HOST_ARCH}-v{{ .KUBERNETES_CNI_VERSION }}.tgz" |
 	sudo tar -C /opt/cni/bin -xz
 
 RELEASE="v{{ .KUBERNETES_VERSION }}"
 
+curl -L https://github.com/kubernetes-sigs/cri-tools/releases/download/${RELEASE}/crictl-${RELEASE}-linux-${HOST_ARCH}.tar.gz |
```

Comment:
There is no 1:1 mapping between kubernetes and cri-tools releases, so I don't think this is correct.

Reply:
Yes, you are right, CRI-tools are released only with the major ones. I'm going to fix it